docs/openapiv2/apidocs.swagger.json

```diff
@@ -719,6 +719,101 @@
+        "operationId": "BatchCheck",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/BatchCheckResponse"
+            }
+          },
+          "400": {
+            "description": "Request failed due to invalid input.",
+            "schema": {
+              "$ref": "#/definitions/ValidationErrorMessageResponse"
+            }
+          },
+          "401": {
+            "description": "Not authenticated.",
+            "schema": {
+              "$ref": "#/definitions/UnauthenticatedResponse"
+            }
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/ForbiddenResponse"
+            }
+          },
+          "404": {
+            "description": "Request failed due to incorrect path.",
+            "schema": {
+              "$ref": "#/definitions/PathUnknownErrorMessageResponse"
+            }
+          },
+          "409": {
```

Comment:
I'm not sure if we have control over this without post-processing, but this is is not possible for BatchCheck

Reply:
That's a good point. How do you remove a specific response type in the swagger definitions? The only customizations I see are some `jq` blocks [in this file](https://github.com/openfga/api/blob/main/scripts/update_swagger.sh#L16)

Reply:
Yeah unfortunately, I think that's the only way for now.

Let's leave that for a separate work, as I think several existing endpoints erroneously show that.